app/views/workspaces/_workspace.html.erb

```diff
@@ -1,7 +1,18 @@
-<li>
-  <%= workspace.name %>
-  <% if in_edit_mode %>
-    <%= link_to 'Show', workspace %>
-    <%= link_to 'Destroy', workspace, method: :delete, data: { confirm: 'Are you sure?' } %>
-  <% end %>
-</li>
+<div class="col-4">
+  <div class="grid-item">
+    <h3><%= link_to workspace.name, workspace %></h3>
```

Comment:
Given that workspace title is currently one single line, we could just use CSS
```css
    text-overflow: ellipsis;
    overflow: hidden;
    white-space: nowrap;
```
Would that be fine?